README.md

```diff
@@ -85,6 +85,38 @@ Run `go get -t ./...` in order to install project's dependencies.
 
 Run tests with `FAUNA_ROOT_KEY="your-cloud-secret" go test ./...`.
 
+Tests can also be run via a Docker container. The `Dockerfile.test` file
+creates an image that automatically runs the tests against a target FaunaDB
+cluster. You can run tests this way via the following steps:
+
+1. Build the image via
```

Comment:
Can we encode these patterns into a Makefile? Or does that become too complicated w/ the number of arguments that we need to provide.

Reply:
I can take a shot at it and see how it goes.